frappe/modules/export_file.py

```diff
@@ -41,6 +41,10 @@ def write_document_file(doc, record_module=None, create_init=True, folder_name=N
 	else:
 		folder = create_folder(module, doc.doctype, doc.name, create_init)
 
+	# override modified_by in json file
+	if not newdoc.custom:
+		newdoc.modified_by = 'Administrator'
```

Comment:
This assumes the `doc` is a "DocType" record. This method is generic to export any `doc` into a JSON. This is also how fixtures work. This logic should be in `doctype.py`.

Reply:
Ok, thanks for explaining.
Will add the simple override on save in doctype.py then 👍